common/docsV1.md

```diff
@@ -249,6 +249,23 @@ Public keys verify that client token requests are coming from an authorized appl
 
 **1. Download the snippet.txt file located in the DPC Portal to create a signature.**
 
+- You are also able to download the snippet.txt file using the command invocation below:
+  <p style="font-weight: 700;">Please refer to this method of downloading the snippet.txt file if you see the error stating <span style="color: #d14">"Unable to verify your public key"</span> in the DPC Portal.</p>
+
+  ~~~
+  curl -JLO https://raw.githubusercontent.com/CMSgov/dpc-app/master/dpc-web/public/snippet.txt 
+  ~~~
+
+**1a. Verify the type of file.**
+
+- If you are using a Mac or Linux shell you can check the file type with this command (will not work in a basic Windows environment)
+
+  ~~~
+  file snippet.txt 
+  ~~~
+  <p style="font-weight: 700;">Response <u>must yield</u> <span style="color: #2A7A3B;">snippet.txt: ASCII text, with no line terminators</span>.</p>
```

Comment:
While I like this addition, it doesn't resolve my above comment.

Reply:
Could you elaborate more on what you are looking for?